implementation/common/src/main/java/io/smallrye/jwt/util/JWTUtilMessages.java

```diff
@@ -2,6 +2,7 @@
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
+import java.security.InvalidAlgorithmParameterException;
```

Comment:
IMHO it should be `IllegalArgumentException` - it is all about the correct algo parameter parameter - it is not happening at the processing time

Reply:
Ignore it please - this is actually a prefect exception type for this case :-)